src/main/webapp/WEB-INF/views/admin/selfCheck/edit-js.jsp

```diff
@@ -47,6 +47,11 @@
 		fn.appendEditVisible($("#append"));
 		fn.initNotice();
 	});
+
+    // close exportlist
+    $(document).click(function(e){
+        if(!$("#ExportContainer").has(e.target).length) $("#ExportList").hide();
```

Comment:
How about changing it like below? It would be more readable.

```js
if(!$("#ExportContainer").has(e.target).length) {
    $("#ExportList").hide();
}
```

Reply:
I fixed it. thank you for review!